src/main/java/org/stocker/routes/UpdateRecordsRoute.java

```diff
@@ -41,6 +45,16 @@ public UpdateRecordsRoute(StockDataClient stockDataClient, StockDBClient stockDB
     @Override
     public void handle(YokeRequest yokeRequest, Handler<Object> handler) {
         YokeResponse response = yokeRequest.response();
+        StockMarket market = null;
+        if (yokeRequest.getParameter("market").equalsIgnoreCase(StockMarket.INDIA.toString())) {
```

Comment:
We will definitely need a middleware which will handle this, I just put an `if..else` right now as a temporary solution